README.rst

```diff
@@ -23,12 +23,7 @@ It can be installed with::
 
 Useful tips:
 
-* Be sure to install it for a version of Python which is compatible
-  with your codebase: for Python 2, ``pip2 install pyflakes`` and for
-  Python3, ``pip3 install pyflakes``.
-
-* You can also invoke Pyflakes with ``python3 -m pyflakes .`` or
-  ``python2 -m pyflakes .`` if you have it installed for both versions.
+* You can invoke Pyflakes with ``python -m pyflakes .``
```

Comment:
I think this can be removed entirely -- or adjusted to use something like `python3.10 -m pip install ...` and `python3.10 -m pyflakes ...`

Reply:
Of the two, I'm maybe team delete since i'd rather not add another thing for maintainers to update in the future

That said, in the `setup.py` it seems like pyflakes always adds some entrypoint, so what if we said:

> You can invoke Pyflakes with `pyflakes .`

Reply:
could use `python3.# ...` so it doesn't need updating again.  the original intent of this was to show how to run it per interpreter -- there isn't really much point in showing `pyflakes .` though

Reply:
`python3.# ...` works for me!

Reply:
Changed my mind, removing seems simpler. Also, the `flake8` part of the section is implied in `Design Principles` so took out the "Useful tips" section

Reply:
this section is important, I want you to adjust it to still make sense but without python 2 -- no reason to throw out the baby with the bathwater

Reply:
okay gave it another shot, appreciate your patience!